src/Optimizely/Utils/EventTagUtils.php

```diff
@@ -38,7 +40,7 @@ public static function getRevenueValue($eventTags) {
             return null;
         }
 
-        if (!isset($eventTags[self::REVENUE_EVENT_METRIC_NAME]) or !$eventTags[self::REVENUE_EVENT_METRIC_NAME]) {
+        if (!isset($eventTags[self::REVENUE_EVENT_METRIC_NAME])) {
```

Comment:
why did u change this method, it means either eventtag doesn't have revenue_event_metric_name or the value inside it is null. 
Please add the condition after OR.

Reply:
Because it failed a test case when I used assertSame in the Event Tag Utils instead of assertEquals (which does type conversion when comparing). 
!$eventTags[self::REVENUE_EVENT_METRIC_NAME]) this condition also returns true when revenue = 0 although 0 is a vlaid value

Reply:
if (!isset($eventTags[self::REVENUE_EVENT_METRIC_NAME])) checks for NULL as well. No need to check for null again.